mne/decoding/time_gen.py

```diff
@@ -896,19 +929,31 @@ class GeneralizationAcrossTime(_GeneralizationAcrossTime):
         Indicates how predictions are achieved with regards to the cross-
         validation procedure:
 
-            ``cross-validation`` : estimates a single prediction per sample
+            * ``cross-validation`` : estimates a single prediction per sample
                 based on the unique independent classifier fitted in the
                 cross-validation.
-            ``mean-prediction`` : estimates k predictions per sample, based on
-                each of the k-fold cross-validation classifiers, and average
+
+            * ``mean-prediction`` : estimates k predictions per sample, based
+                on each of the k-fold cross-validation classifiers, and average
                 these predictions into a single estimate per sample.
 
-        Default: 'cross-validation'
+        Defaults to 'cross-validation'.
     scorer : object | None | str
-        scikit-learn Scorer instance or str type indicating the name of
-        the scorer such as `accuracy`, `roc_auc`. If None, set to 
-        accuracy_score.
+        scikit-learn Scorer instance or str type indicating the name of the
+        scorer such as ``accuracy``, ``roc_auc``. If None, set to ``accuracy``.
+    score_mode : {'fold-wise', 'mean-fold-wise', 'sample-wise'}
+        Determines how the scorer is estimated:
+
+            * ``fold-wise`` : returns the score obtained in each fold.
+
+            * ``mean-fold-wise`` : returns the average of the fold-wise scores.
 
+            * ``sample-wise`` : returns score estimated across across all
+                y_pred independently of the cross-validation. This method is
+                faster than ``mean-fold-wise`` but less conventional, use at
+                your own risk.
```

Comment:
I think we had already agreed on the labels above, the semantics now seem to be swapped.
Implicitly these labels will tell you if the output is 2d or 3d, this should be consistent. Jona would you be fine with either of the 2 swapping proposals above?